test/module/irohad/network/block_loader_test.cpp

```diff
@@ -52,15 +60,30 @@ class BlockLoaderTest : public testing::Test {
     builder.RegisterService(service.get());
     server = builder.BuildAndStart();
 
-    peer = Peer();
+    Peer peer;
     peer.address = "0.0.0.0:" + std::to_string(port);
+    std::copy_n(
+        peer_key.blob().begin(), peer.pubkey.size(), peer.pubkey.begin());
     peers.push_back(peer);
 
     ASSERT_TRUE(server);
     ASSERT_NE(port, 0);
   }
 
-  Peer peer;
+  auto base() const {
+    constexpr auto kTotal = (1 << 5) - 1;
+    return shared_model::proto::TemplateBlockBuilder<
+               kTotal,
+               shared_model::validation::DefaultBlockValidator,
+               shared_model::proto::Block>()
+        .txNumber(0)
+        .height(0)
```

Comment:
you always set it to 1 further. mb it is better to set here?

Reply:
Not at the all cases, look [here](https://github.com/hyperledger/iroha/pull/949/files/3405edd8544cf05eece72ebf64bbc0ffbf99eee9#diff-4ec25e0d6a030bb199ddde4f4a681be6R170) for example

Reply:
upd: doesn't seem necessary, I'll fix that as well